notebooks/Linear Regression.ipynb

```diff
@@ -0,0 +1,1288 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style>\n",
+       "    .dataframe thead tr:only-child th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: left;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Ca</th>\n",
+       "      <th>P</th>\n",
+       "      <th>pH</th>\n",
+       "      <th>SOC</th>\n",
+       "      <th>Sand</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>-0.295749</td>\n",
+       "      <td>-0.041336</td>\n",
+       "      <td>-1.129366</td>\n",
+       "      <td>0.353258</td>\n",
+       "      <td>1.269748</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>-0.387442</td>\n",
+       "      <td>-0.231552</td>\n",
+       "      <td>-1.531538</td>\n",
+       "      <td>-0.264023</td>\n",
+       "      <td>1.692209</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>-0.248601</td>\n",
+       "      <td>-0.224635</td>\n",
+       "      <td>-0.259551</td>\n",
+       "      <td>0.064152</td>\n",
+       "      <td>2.091835</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>-0.332195</td>\n",
+       "      <td>-0.318014</td>\n",
+       "      <td>-0.577548</td>\n",
+       "      <td>-0.318719</td>\n",
+       "      <td>2.118477</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>-0.438350</td>\n",
+       "      <td>-0.010210</td>\n",
+       "      <td>-0.699135</td>\n",
+       "      <td>-0.310905</td>\n",
+       "      <td>2.164148</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         Ca         P        pH       SOC      Sand\n",
+       "0 -0.295749 -0.041336 -1.129366  0.353258  1.269748\n",
+       "1 -0.387442 -0.231552 -1.531538 -0.264023  1.692209\n",
+       "2 -0.248601 -0.224635 -0.259551  0.064152  2.091835\n",
+       "3 -0.332195 -0.318014 -0.577548 -0.318719  2.118477\n",
+       "4 -0.438350 -0.010210 -0.699135 -0.310905  2.164148"
+      ]
+     },
+     "execution_count": 78,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "y.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why Linear Regression?"
```

Comment:
I see a bunch of changes -- this comment just isn't being marked as out-dated because it's on the title.